src/components/MGN/BalanceDisplays.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import Countdown from './Countdown'
+import { MGNbalances } from './helpers'
+import { BigNumber } from 'types'
+
+const addressRexp = '^\\s*0x[0-9A-Fa-f]{40}\\s*$'
+
+interface AccountBalanceProps {
+  balances: MGNbalances;
+  now: number;
+  account?: string,
+}
+
+const toMGNnumber = (bn: BigNumber | null) => bn && bn.div(10 ** 18).toFixed(4)
```

Comment:
this will eval to true if passed in sth else, idk if there is any case when a BN type is not passed in but there is also an `isBN` method available if that is useful

Reply:
That's what TS is for, no? Only BN | null accepted